### PR TITLE
Bump to version 1.0.0

### PR DESCRIPTION
**Pull request by marcotc**
🎉

### CODE REVIEW

**Review by lloeki — APPROVED** [review]

File: gemfiles/ruby_3.2.0_contrib_old.gemfile [hunk 1]
```diff
@@ -37,5 +37,6 @@ gem "google-protobuf", ["~> 3.0", "!= 3.7.0", "!= 3.7.1"]
 gem "sorbet", "= 0.5.9672"
 gem "spoom", "~> 1.1"
 gem "dalli", "< 3.0.0"
+gem "graphql", ">= 1.12.0", "< 2.0"
```
Comment (lloeki) [inline gemfiles/ruby_3.2.0_contrib_old.gemfile:40]: Just wondering if this was expected in this changeset.

File: gemfiles/ruby_3.2.0_contrib.gemfile [hunk 2]
```diff
@@ -49,7 +49,7 @@ gem "elasticsearch-transport"
 gem "ethon"
 gem "excon"
 gem "grape"
-gem "graphql", ">= 1.12.0", "< 1.13.0"
+gem "graphql", ">= 2.0"
```
Comment (lloeki) [inline gemfiles/ruby_3.2.0_contrib.gemfile:52]: Just wondering if these were expected in this changeset.

Comment (ivoanjo) [reply gemfiles/ruby_3.2.0_contrib.gemfile:52]: I can explain this one:

1. https://github.com/DataDog/dd-trace-rb/pull/1974 and https://github.com/DataDog/dd-trace-rb/pull/1982 were opened concurrently
2. I merged https://github.com/DataDog/dd-trace-rb/pull/1974 first
3. I suggested Marco not to bother updating the gemfiles/ for 3.2 as anyway they don't run in CircleCI yet (https://github.com/DataDog/dd-trace-rb/pull/1982#discussion_r858432943)
4. The changes showed up here when Marco run the command to update all gemfiles

So this is 100% harmless :)

File: gemfiles/ruby_3.2.0_contrib_old.gemfile (hunk 1 above)
Comment (ivoanjo) [reply gemfiles/ruby_3.2.0_contrib_old.gemfile:40]: Same as https://github.com/DataDog/dd-trace-rb/pull/1993#discussion_r861551459 😄

**Pull request merged**